src/ecc/codes/surface.jl

```diff
@@ -0,0 +1,60 @@
+"""The planar surface code refers to the toric code [kitaev2003fault](@cite) in a 2D lattice with open boundaries.
+
+Illustration of a 3x2 surface code, where qubits are located on the edges:
```

Comment:
```suggestion
Illustration of a 3×2 surface code, where qubits are located on the edges:
```

Minor typography polish, using the unicode "times" symbol instead of the letter x.